docs/en/ingest-management/fleet-overview.asciidoc

```diff
@@ -27,6 +27,16 @@ within seconds.
 [role="screenshot"]
 image::images/integrations.png[Integrations page]
 
+[discrete]
+[[fleet-server]]
+== {fleet-server}
+
+Fleet-server is the mechanism to connect {agent}s to {fleet}.  It allows for
+a scalable infrastructure and is supported in Elastic Cloud and self-manage clusters.
+Fleet-server is a separate process that runs which communicates with the {agent}s
+deployed.  It can be started from any of the x64 architecture {agent} artifacts
+available.
```

Comment:
```suggestion
{fleet-server} is a separate process that communicates with the deployed {agent}s.
It can be started from any available x64 architecture {agent} artifact.
```

Reply:
This begs the question: are there Elastic Agent artifacts that don't support Fleet Server?

Reply:
(I think you could even remove "available" because that seems implicit)

Reply:
the x86 artifacts do note - there are several:  for x86 .tar.gz, x86 deb/rpm and x86 windows